objects/Cargo.toml

```diff
@@ -35,6 +35,9 @@ vm-core = { workspace = true }
 vm-processor = { workspace = true }
 winter-rand-utils = { version = "0.9", optional = true }
 
+[target.'cfg(target_arch = "wasm32")'.dependencies]
```

Comment:
minor: when I was using these to gate certain crates for wasm builds in the `miden-client`, the behavior was a bit finnicky. Not feedback for this PR, but if we see this start to become fragile for whatever reason, switching to features might help solve it.